modes/magit/evil-collection-magit.el

```diff
@@ -342,7 +343,7 @@ moment.")
            (,states magit-mode-map "yr"   magit-show-refs            "y")
            (,states magit-mode-map "ys"   magit-copy-section-value   "C-w")
            (,states magit-mode-map "yb"   magit-copy-buffer-revision "M-w")
-           ((visual) magit-mode-map "y"   evil-yank))
+           ((visual) magit-mode-map "y"   magit-copy-section-value))
```

Comment:
This seems like it might be overkill, I'm imagining in magit-mode, there might be reasons to use evil-yank (even in visual state). Maybe I'm misunderstanding everything magit-copy-section-value can do. As I understand, ys should be bound to that function right?

Reply:
`ys` is a motion key which is not proper in such a case

Reply:
> This seems like it might be overkill, I'm imagining in magit-mode, there might be reasons to use evil-yank (even in visual state). Maybe I'm misunderstanding everything magit-copy-section-value can do. As I understand, ys should be bound to that function right?

`ys` is a motion key which is not proper in such a case

Reply:
What does "motion key" mean in this case? example?

For example:

```
y s runs the command magit-copy-section-value, which is an autoloaded
interactive Lisp function in ‘magit-extras.el’.

It is bound to y s, <normal-state> y s.

(magit-copy-section-value ARG)
```

Reply:
Concerning the overkill-argument, `magit-copy-section-value` without a prefix-arg simply forwards to `copy-region-as-kill` if a transient-region is active (same as visual state?). However, `evil-yank` is different from `copy-region-as-kill`, so if we don't want to shadow it completely, we'd need to use a different binding, like for example the suggested `ys`, and keep `evil-yank` on `yy` or similar.

One point where `evil-yank` would be necessary is in block-mode, in order to properly yank the selected block. However, horizontal navigation with `h` and `l` is anyway not possible in magit-buffers (it is still possible with cursor keys...). So I'd raise the question if it is ok to accept the forwarding to `copy-region-as-kill` and shadow `evil-yank` completely.

But I'd also be happy with the first approach to have different bindings.

Reply:
OK, if that's true it might be OK to use magit-copy-section-value. Lemme find some time to pull your PR and play with it myself to see if there are any extra subtleties.